tool/state-processor/src/type_registry.rs

```diff
@@ -3,6 +3,8 @@ use parity_scale_codec::{Decode, Encode};
 
 pub const GWEI: u128 = 1_000_000_000;
 pub const KTON_ID: u64 = 1026;
+https://github.com/darwinia-network/darwinia-2.0/blob/main/runtime/common/src/gov_origin.rs#L46
```

Comment:
Format & better use a fixed link. The main branch is not fixed.

Reply:
Updated the link with commit info instead of the branch. It's a fixed link in my mind. Please review it again.